kinesis/src/main/scala/akka/stream/alpakka/kinesis/KinesisFlowErrors.scala

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.kinesis
+
+import com.amazonaws.services.kinesis.model.PutRecordsResultEntry
+
+import scala.util.control.NoStackTrace
+
+object KinesisFlowErrors {
```

Comment:
Done.

About the settings, I'd rather have them separated, it could be a mess mixing all those companion objects, validations and constructors. And think that we may have more Kinesis Sources/Flows in the future with different settings.